cli/docker/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+services:
+  hookdeck:
+    image: hookdeck/hookdeck-cli:latest
+    container_name: hookdeck
+    entrypoint: ["/bin/sh","-c"]
+    env_file:
+      - path: ./.env
+        required: true
+    command:
+      - |
+        /bin/hookdeck version
+        /bin/hookdeck ci --api-key $HOOKDECK_API_KEY
+
+        echo "Installing curl dependency. This could be done within a Dockerfile."
+        apk add curl
+
+        echo "Ensuring connection exists for source $HOOKDECK_SOURCE_NAME"
+        curl --location --request PUT 'https://api.hookdeck.com/2024-03-01/connections' \
+          --header 'Content-Type: application/json' \
+          --header 'Accept: application/json' \
+          --header 'Authorization: Bearer $HOOKDECK_API_KEY' \
+          --data '{
+                    "name": "$HOOKDECK_SOURCE_NAME_to_CLI",
```

Comment:
HOOKDECK_SOURCE_NAME_to_CLI isn't mentioned in the env file above

Reply:
`HOOKDECK_SOURCE_NAME` is the env var. Maybe I should put curly brackets around env var.

`${HOOKDECK_SOURCE_NAME}_to_CLI`